airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -37,16 +41,54 @@ fi
 echo "Mounting data partition"
 mount /dev/sda3 /mnt
 
-echo "Decompressing and writing:"
-lz4 -c -d /mnt/image.img.lz4 | pv -s 50g > /dev/nvme0n1 
-#lz4 -c -d /usr/share/vx-img/image.img.lz4 | pv -s 50g > /dev/nvme0n1 
+_path="/mnt"
+_supported=('gz' 'lz4')
+_hashash=0
+
+_toflash=""
+for f in $_path/*; do
+    _filename="${f##*/}"
+    _extension="${_filename##*.}"
+
+    _compression=""
+    if [[ "$_extension" == "gz" ]]; then
+        _compression="gzip";
+    elif [[ "$_extension" == "lz4" ]]; then
+        _compression="lz4";
+    elif [[ "#_extension=sha256sum" ]]; then
+        _hashash=0
+    fi
+    
+    if [ ! -z "$_compression" ]; then
+        echo "Found $f, extract it using $_compression and flash? [y/n]"
```

Comment:
So this is what you were saying how it would loop through each option asking? Lets make it a menu like the other one where it lists all the options and lets you chose. IDK if this is doing this or not but I think its fine to just only look in the top directory and not recursively